api/test-helpers.js

```diff
@@ -175,6 +220,7 @@ module.exports = {
   BaseFacade: BaseFacade,
   makeBakery: makeBakery,
   makeConnection: makeConnection,
+  makeConnectionWithResponse: makeConnectionWithResponse,
   makeWSClass: makeWSClass,
   requestEqual: requestEqual
 };
```

Comment:
Pre-existing I know, but maybe we should simplify all of those.

Reply:
updated